syntax = "proto3";

service HelloWorld {
    rpc SayHello (HelloRequest) returns (HelloReply);
}

service SumService {
    rpc Summation (SumRequest) returns (SumReply);
}

message HelloRequest {
    string name = 1;
}

message HelloReply {
    string message = 1;
}

message SumRequest {
    int32 a = 1;
    int32 b = 2;
}

message SumReply {
    int32 result = 1;
}


